test/e2e/playwright/mmi/specs/dapp.spec.ts

```diff
@@ -18,6 +18,7 @@ const dappsTest = async (
   await mainPage.bringToFront();
   await mainPage.openActivityTab();
   await mainPage.checkLastTransactionStatus(/created/iu);
+  await page.waitForTimeout(2000);
```

Comment:
Why do we need this timeout of 2000?

Reply:
Sometimes we dont have a value in the `data-custodiantransactionid` from the getCustodianTXId fast enough.

Reply:
I'm not very familiar with Playwright, but don't they have a method similar to `waitForSelector`?